src/basic/calendarspec.c

```diff
@@ -349,9 +352,7 @@ int calendar_spec_to_string(const CalendarSpec *c, char **p) {
         fputc_unlocked(':', f);
         format_chain(f, 2, c->microsecond, true);
 
-        if (c->utc)
-                fputs_unlocked(" UTC", f);
-        else if (IN_SET(c->dst, 0, 1)) {
+        if (IN_SET(c->dst, 0, 1)) {
```

Comment:
i am surprised about the ordering here, shouldn't the explicit timezone if it is configured be shown instead of the implicit one here?

Also, I think it would make sense to continue to support UTC explicitly. It is after all a very special timezone, probably pretty regularly used and the only one besides the local one which we can support without doing the fork() magic. Hence, let's support calendar in three different modes: in the local timezone, in UTC and in any other timezone.